composer.json

```diff
@@ -45,7 +45,8 @@
         "vlucas/phpdotenv": "^2.4",
         "webflo/drupal-finder": "^1.0.0",
         "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.0"
+        "zaporylie/composer-drupal-optimizations": "^1.0",
+        "ext-json": "*"
```

Comment:
Remove this and use this instead: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Serialization%21Json.php/class/Json/8.2.x

Reply:
Done